### ISSUE TITLE
[Bug]: Redis extension missing in PHP 8.4.0alpha1

### ISSUE THREAD

**Issue by ju5t** [beyondcode/herd-community#938]
### Platform

macOS

### Operating system version

macOS 14.5

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.9.1 Build 31

### PHP Version

8.4.0alpha1

### Bug description

It seems that the 'redis' extension is missing in PHP 8.4. Although we don't use Redis, Laravel 11 doesn't like that it's not there. Perhaps it's also possible to update PHP 8.4 to one of the beta releases?

### Steps to reproduce

```shell
php -m
```

### Relevant log output

```
[PHP Modules]
bcmath
bz2
calendar
Core
ctype
curl
date
dba
dom
exif
FFI
fileinfo
filter
ftp
gd
gmp
hash
iconv
igbinary
intl
json
ldap
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
pdo_pgsql
pdo_sqlite
pgsql
Phar
posix
random
readline
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
sqlite3
standard
sysvmsg
sysvsem
sysvshm
tokenizer
xml
xmlreader
xmlwriter
xsl
zip
zlib
zstd
```

**Comment by mpociot** [comment]
The Redis PHP extension is not yet available, as PHP 8.4 is still an alpha release.

**Comment by ju5t** [comment]
@mpociot there does seem to be a 6.1.0 RC for Redis which I think supports PHP 8.4.

> as PHP 8.4 is still an alpha release.

The latest version is beta3. But if it's too much work I'll wait until 8.4 is out.
